ide/hudson/src/org/netbeans/modules/hudson/Bundle.properties

```diff
@@ -17,7 +17,7 @@
 
 OpenIDE-Module-Display-Category=Base IDE
 OpenIDE-Module-Long-Description=\
-    Helps browse and manage installations of the Hudson continuous integration server (http://hudson-ci.org/). \
+    Helps browse and manage installations of the Jenkins continuous integration server (http://hudson-ci.org/). \
```

Comment:
Should we change the URL to https://www.jenkins.io/ ?

Reply:
I thinks this would be nice